jekyll/_cci2/projects.md

```diff
@@ -11,7 +11,7 @@ contentTags:
 ---
 
 
-A CircleCI project shares the name of the associated code repository in your [version control system]({{ site.baseurl }}/gh-bb-integration/) (VCS). Select **Projects** in the CircleCI web app sidebar to enter the projects dashboard. From here you can set up and follow the projects you have access to.
+A CircleCI project shares the name of the associated code repository in your [version control system]({{ site.baseurl }}/github-integration/) (VCS). Select **Projects** in the CircleCI web app sidebar to enter the projects dashboard. From here you can set up and follow the projects you have access to.
```

Comment:
@rosieyohannan I have done this replacement on my own. Can you please make sure this is accurate? thanks

Reply:
I was going to suggest we just remove the link altogether here but this page is actually about to be archived! (after we get this PR merged though)